bridgetown-core/lib/bridgetown-core/concerns/site/writable.rb

```diff
@@ -20,6 +20,11 @@ def write
       Bridgetown::Hooks.trigger :site, :post_write, self
     end
 
+    # Yields the pages from {#pages} {#static_files} and {#docs_to_write}
+    # @yieldparam item [Page] Yields a {#Bridgetown::Page} Object to be manipulated.
```

Comment:
I think it's a few different classes that get yielded…I'm guessing Page, Document, and StaticFile.

Reply:
I wasn't sure about this one. I think Page May be a superclass but I'm not sure tbh. I'll have to revisit this.

Reply:
You're right. It yields all 3. Will update it accordingly.